cpu/native/can/candev_linux.c

```diff
@@ -195,7 +195,11 @@ static int _init(candev_t *candev)
 
     real_bind(dev->sock, (struct sockaddr *)&addr, sizeof(addr));
 
-    _set_bittiming(dev, &candev->bittiming);
+    /* Only set bitrate on real can interfaces.
+     * Not supported on virtual can interfaces ("vcanX") */
+    if (!strncmp(dev->conf->interface_name, "can", strlen("can"))) {
+        _set_bittiming(dev, &candev->bittiming);
```

Comment:
Maybe add
```C
else {
    LOG_INFO("not setting bitrate on virtual can interface %s\n", dev->conf->interface_name);
}
```

so if `udev` devices can devices should now follow the same naming scheme as network devices, there is at least a chance of finding out why it broke 😉

Reply:
👍 and maybe change the logic to match the one below in `_set`, i.e. 

```
if (strncmp(dev->conf->interface_name, "can", strlen("can"))) {
    LOG_INFO(...);
}
else {
    _set_bittiming(dev, &candev->bittiming);
}
```